### PR TITLE
pt-br translation

### PR DESCRIPTION
**Pull request by brccabral**
Hi,
I would like to make suggestions in the Portuguese Brazil translations, they were not totally wrong, but some were out of context or they were Portuguese Portugal. From a Brazilian perspective, this makes more sense to me. Also, some cosmetic changes.

Bruno

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -30,3 +30,4 @@ Makefile.in
 /flatpak/.flatpak-builder/
 /flatpak/build-dir/
 /.cache/
+/.vscode/
```
Comment (bk138) [inline .gitignore:33]: Unrelated, please remove.

File: po/pt_BR.po [hunk 2]
```diff
@@ -39,7 +39,8 @@ msgid ""
 msgstr ""
 "Gostaria de mostrar esta introdução novamente na próxima vez que iniciar o "
 "Gromit-MPX? \n"
-"Você sempre pode acessar novamente através do menu de bandeja do sistema.\n"
+"Você sempre pode acessar novamente através da área de notificação do "
```
Comment (bk138) [inline po/pt_BR.po:42]: Hm, seems to me this is not really true; the icon is not in the notification area, at least not for GNOME. In Gnome, the notification area is this https://tse1.mm.bing.net/th?id=OIP.N4gSt_FNwPdNqiVUAq6R1QHaEo&pid=Api as opposed to the tray which is optional but appears like this: https://i.ytimg.com/vi/JoodcJdfmJQ/maxresdefault.jpg

**Review by bk138 — CHANGES_REQUESTED** [review]

File: po/pt_BR.po (hunk 2 above)
Comment (bk138) [reply po/pt_BR.po:42]: Pinging the initial translator @natenho: what do you think? If you both as native speakers agree the new wording is better, I'll merge it :-).

File: .gitignore (hunk 1 above)
Comment (brccabral) [reply .gitignore]: removed

File: po/pt_BR.po (hunk 2 above)
Comment (natenho) [reply po/pt_BR.po:42]: Hello! Both translations are acceptable, "área de notificação" is the translation currently used by Microsoft in Windows although It used to be "bandeja do sistema" for some time. 
 (https://support.microsoft.com/pt-br/windows/personalizar-a-%C3%A1rea-de-notifica%C3%A7%C3%A3o-da-barra-de-tarefas-e159e8d2-9ac5-b2bd-61c5-bb63c1d437c3#WindowsVersion=Windows_10). 

I personally don't like Microsoft's choice.

Comment (bk138) [reply po/pt_BR.po:42]: KDE are [using](https://kde.org/pt-br/plasma-desktop/) "bandeja do sistema" 
grafik so I would rather keep it to what Linux users are used to. I'll not merge this one but want to say thanks for the time you put in @brccabral !

**Pull request closed without merging**